src/utils.mli

```diff
@@ -55,6 +55,18 @@ val install_file
   -> findlib_toolchain:string option
   -> string
 
+module type Persistent_desc = sig
+  type t
+  val name : string
+  val version : int
+end
+
+(** Persistent value stored on disk *)
+module Persistent(D : Persistent_desc) : sig
+  val dump : Path.t -> D.t -> unit
+  val load : Path.t -> D.t option
+end
+
```

Comment:
Imo, having this stuff live in its own module is cleaner.

Reply:
Possibly, and maybe `Cached_digest` as well. We are just starting to have a lot of files in src/...